handbook/engineering/search/iteration_log.md

```diff
@@ -25,9 +25,13 @@ This document contains the goals and work log for the search team's [2-week iter
 
 ### Streaming search
 
-- **Owner(s):** Keegan, TODO frontend engineer
+- **Owner(s):** Keegan, Juliana
 - **Outcomes:**
-    - $DESIRED_OUTCOME
+    - Make a decision on whether to integrate with existing search results components or write a new set of components to show streamed results.
+    - Identify which metatada we want to display while streaming results, and define corresponding API contract.
+    - Define experience for large result sets (how many results are displayed initially, "show more" versus infinite scrolling).
+    - Define experience for dynamic filters.
+    - We have a project board representing remaining design + implementation work for future iterations.
```

Comment:
@keegancsmith @limitedmage @rrhyne PTAL. Are you comfortable with these desired outcomes for our current iteration ending 2020-10-02? Are we missing anything?

Reply:
sounds good. There is things I could be doing in the backend, but I'd prefer to spend that time on performance work and plan it once its very clear what our contract between backend and frontend is.

Reply:
I think 3.21 could be feasible for all these outcomes, but since this requires a lot of collaboration with design team, it is heavily dependent on design bandwidth.

Reply:
@rrhyne do you think you will have the bandwidth to help with the above outcomes between now and 2020-10-02?

Reply:
@rrhyne I'll merge this as-is, let's chat about this live today and edit expectations if needed.

Reply:
Yes, product planned on me supporting this effort.